doc/ref/states/top.rst

```diff
@@ -64,304 +134,224 @@ just the state tree, then the :conf_master:`file_roots` option can be expanded:
       prod:
         - /srv/salt/prod
 
-Then our top file could reference the environments:
+In the above, we declare three environments: `dev`, `qa` and `prod`.
+Each environment has a single directory assigned to it.
+
+Our top file references the environments:
 
 .. code-block:: yaml
 
     dev:
-      'webserver*dev*':
+      'webserver*':
```

Comment:
Good point. I ended up removing these because I felt like they were making the documentation too difficult to understand.